src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -153,14 +157,11 @@ def run_sequence
       def find_products
         log.info "Loading installed products"
 
-        if !SwMgmt.init
+        if !SwMgmt.init(true)
```

Comment:
I prefer here to use exception, so problem with ignored return value below cannot happen